### DISCUSSION TITLE
[Jetbrains PhpStorm] Copilot Login not working, ENOTCONN, error code: -32603

### DISCUSSION THREAD

**Discussion by Maybach91** [community/community#41505] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I’m using PhpStorm 2022.2.4 and the latest Plugin Version. But when I want to log in to GitHub Copilot, I get the following popup:

`Request signInInitiate failed with message: read ENOTCONN, request id: 3, error code: -32603`

<img width="416" alt="image" src="https://user-images.githubusercontent.com/1867543/207044296-5c44b754-e758-4a08-bc6a-174d83515bc0.png">

There is no VPN or corporate network firewall used. Also, no proxy is used. All these solutions:

1. https://github.com/community/community/discussions/35544
2. https://github.com/community/community/discussions/40882
3. https://github.com/community/community/discussions/40896
4. https://github.com/djoxy-code/Github-copilot-Fix
5. https://stackoverflow.com/questions/72505280/idea-plugin-github-copilot-cant-login-in-github/72564971#72564971
6. https://stackoverflow.com/questions/73639787/unable-to-use-github-copilot-on-webstorm-jetbrains-when-behind-corporate-proxy

Already provided are not working since they just state the proxy thing. Is there ANY fix/workaround? 

EDIT: solved:  my network filter (little snitch) blocked the connection

**Comment by siddharthgoel88** [comment]
Check this out https://sidd.io/2023/01/github-copilot-self-signed-cert-issue/

**Reply by zealrussell** [reply]
Thank you, this works!

**Reply by aligajani** [reply]
Still doesn't work on Mac OS , using latest Phpstorm
